.github/actions/pr-comment/action.yml

```diff
@@ -5,6 +5,9 @@ inputs:
   name:
     description: 'Artifact name to import comment data from.'
     required: true
+  mode:
+    description: 'Mode of operation (upsert/recreate/delete).'
+    required: true
```

Comment:
Variable needs to be forwarded to the `thollander/actions-comment-pull-request` action, no? Or is this somehow happening automatically?

Reply:
Doh. Of course.